### ISSUE TITLE
data preparation with python

### ISSUE THREAD

**Issue by twtygqyy** [huangzehao/caffe-vdsr#17]
Hi, I tried to prepare the data with cv2 resize function instead of matlab imresize, the training loss is not bad but the psnr performance is far worse than the paper. Any idea about that? Does it only due to the difference between opencv and matlab?

**Comment by huangzehao** [comment]
Hi, the performance of cv2 resize function in opencv is worse than matlab imresize. Check this [discussion](http://stackoverflow.com/questions/27851389/opencv-image-resize-comparison-to-matlab).
You can try to use the matlab imresized image as input to test your model (or use cv2 resized image to test my model), for finding out the problem. BTW, maybe scripy.misc.imresize is a good choice, you can have a try.

**Comment by twtygqyy** [comment]
@huangzehao thx for the reply, I also tried scripy.misc.imresize, the performance was a little bit better than opencv's function, but still much worse than Matlab. I will follow your suggestion and had a test. Thx again

**Comment by huangzehao** [comment]
Close the issue. You can reopen it if you have any new progress.
